### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by LaskoA**
Please check if everything is OK so far. If yes, I will continue with Extra section. Thank you ;)

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,26 @@
...
+    1: 0.0,
+    2: 0.0,
+    3: 0.0,
+    4: 0.0,
+    5: 0.0,
+    6: 0.0,
+    7: 0.0,
+    8: 0.0,
+    9: 0.0,
+    10: 0.0}
```
Comment (Abnormaltype) [inline app/main.py:13]: It is bad style to write dictionary like this manually.

File: app/main.py [hunk 2]
```diff
@@ -1 +1,26 @@
...
+option = ["heads", "tails"]
+
+
+def flip_coin():
+    for case in range(0, 10000):
+        for num, flip in enumerate(range(1, 11)):
+            flip = random.choice(option)
+            if flip == "heads":
```
Comment (Abnormaltype) [inline app/main.py:21]: and where condition when `flip =="tails"`?

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Hi :) 

1) I have changed the way of adding elements to dictionary, please check it now;
2) Do we really need to count number of "tails"? It`s not mentioned in the task at all. Or did I miss something?
3) Added "draw_gaussian_distribution_graph" function - please check it as well.

Thank you.

**Review by IvanRamyk — CHANGES_REQUESTED** [review]
I tried to execute `flip_coin` function and got an error
Could you call your functions from the module and make sure they work correctly?

**Comment by LaskoA** [comment]
Problem was fixed (line 14, dictionary[num] instead of num). Kindly ask you to check again. Thank you :)

**Review by IvanRamyk — APPROVED** [review]

**Review by Abnormaltype — APPROVED** [review]

**Pull request closed without merging**